### DISCUSSION TITLE
Bad UX due to large unoptimized video size

### DISCUSSION THREAD

**Discussion by janviehweger** [iotaledger/firefly#1631] [Features / Enhancements]
When accessing the Firefly wallet on a low bandwidth network, I was unable to access the migration video. The three-minute short clip is 240MB!!! big, which is definitely way too much. It doesn't seem to be optimized at all. I was able to sample the video down with handbrake to 29mb at 1080p without any loss of quality. 720p would be sufficient also to my opinion.

The video:
https://d17lo1ro77zjnd.cloudfront.net/firefly/videos/ledger_integration_v12.mp4

https://github.com/iotaledger/firefly/blob/d3d21b287e97baf8a3069a229dc08ebe3cbde456/packages/desktop/public/index.html#L13

**Comment by rafaeldjpbrochado** [comment]
@rajivshah3 has implemented this change (https://github.com/iotaledger/firefly/issues/1806). Thanks for suggesting it!
